Makefile

```diff
@@ -262,6 +290,46 @@ else
 	$(error addlicense must be installed for this rule: go get -u github.com/google/addlicense)
 endif
 
+### gen_csv: generate the csv for a newer version
+gen_csv: update_devworkspace_crds update_terminal_manifests
+	operator-sdk generate csv --apis-dir ./pkg/apis --csv-version 1.0.0 --make-manifests --update-crds --operator-name "web-terminal" --output-dir ./web-terminal-operator
+	
+	# filter the deployments so that only the valid deployment is available. See: https://github.com/eclipse/che/issues/17010
+	cat ./web-terminal-operator/manifests/web-terminal.clusterserviceversion.yaml | \
+	yq -Y \
+	'.spec.install.spec.deployments[] |= select( .spec.selector.matchLabels.app? and .spec.selector.matchLabels.app=="che-workspace-controller")' | \
+	tee ./web-terminal-operator/manifests/web\ terminal.clusterserviceversion.yaml >>/dev/null
+
+	cp devworkspace-crds/deploy/crds/workspace.devfile.io_devworkspaces_crd.yaml web-terminal-operator/manifests
+
+### olm_build_bundle: build the bundle that will be stored on quay
+olm_build_bundle: _print_vars
+	# Create the bundle and push it to quay
```

Comment:
it is always quay or users are able to configure any docker registry like dockerhub with BUNDLE_IMG param?